pulp_ansible/app/models.py

```diff
@@ -275,6 +318,381 @@ class Meta:
         unique_together = ("pubkey_fingerprint", "signed_collection")
 
 
+
+class SigstoreSigningService(Content):
```

Comment:
This should not be content.

Reply:
What would be a better model to subclass?

Reply:
SigningService or BaseModel